### DISCUSSION TITLE
Getting error as 'cannot use import statement outside a module for remark-gfm'

### DISCUSSION THREAD

**Discussion by venumanthena** [remarkjs/remark#807] [Q&A] [answered]
Hi,
Thanks for the great plugin.
I succesfully installed react-markdown. However after installing the remark-gfm plugin via npm, I am getting error as 'cannot use import statement outside module'. I don't get these errors for other packages.  What am I doing wrong here? Thanks.

**Comment by heeyio** [comment]
Hi! 
Same.

**Comment by ChristianMurphy** [comment, marked as answer]
Hey @venumanthena :wave: 
Sorry you ran into a spot of trouble.
It sounds like you are using `remark-gfm` version 2 with `react-markdown` version 6?

If so two things:
1. `react-markdown` version 6 isn't compatible with `remark-gfm` version 2, hold on `remark-gfm` version 1 for now (see: https://github.com/remarkjs/react-markdown/issues/629)
2. `remark-markdown` version 7 is incoming soon, and will support `remark-gfm` version 2. Part of `remark-gfm` version 2 is that it uses [ESModule packaging](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules), the error you are seeing "cannot use import statement outside module" looks to be related to a tool not being configured to support ESM, this guide from Sindre provides advise on how to use ESM with a number of build tools https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c

**Reply by venumanthena** [reply]
Thanks Christian :)
